website/src/components/SequenceDetailsPage/SequencesDisplay/getSequenceNames.tsx

```diff
@@ -1,34 +1,33 @@
-import { type ReferenceGenomesSequenceNames, SINGLE_REFERENCE } from '../../../types/referencesGenomes.ts';
+import { type ReferenceGenomesLightweightSchema, SINGLE_REFERENCE } from '../../../types/referencesGenomes.ts';
 import {
     getMultiPathogenNucleotideSequenceNames,
     getMultiPathogenSequenceName,
     getSinglePathogenSequenceName,
     isMultiSegmented,
-    type SequenceName,
+    type SegmentOrGeneInfo,
 } from '../../../utils/sequenceTypeHelpers.ts';
 
 export function getSequenceNames(
-    referenceGenomeSequenceNames: ReferenceGenomesSequenceNames,
+    referenceGenomeLightweightSchema: ReferenceGenomesLightweightSchema,
     suborganism: string,
 ): {
```

Comment:
Sometime we should probably define a type for what this returns - either now or make an issue? We should probably call this `subOrganismLightweightSchema` or something, using the same logic I mentioned that "names" implies a type of `String []`